src/sdk/sql_cluster_router.cc

```diff
@@ -2315,11 +2315,32 @@ bool SQLClusterRouter::IsEnableTrace() {
 }
 
 ::hybridse::sdk::Status SQLClusterRouter::SetVariable(hybridse::node::SetPlanNode* node) {
-    if (node->Scope() == hybridse::node::VariableScope::kGlobalSystemVariable) {
-        return {::hybridse::common::StatusCode::kCmdError, "global system variable is unsupported"};
-    }
     std::string key = node->Key();
     std::transform(key.begin(), key.end(), key.begin(), ::tolower);
+    if (node->Scope() == hybridse::node::VariableScope::kGlobalSystemVariable) {
+        std::lock_guard<::openmldb::base::SpinMutex> lock(mu_);
+        std::string value = node->Value()->GetExprString();
+        std::transform(value.begin(), value.end(), value.begin(), ::tolower);
+        if (key != "execute_mode" && key != "enable_trace") {
+            return {::hybridse::common::StatusCode::kCmdError, "no global variable " + key};
```

Comment:
`execute_mode` and `enable_trace` is session variables

Reply:
> `execute_mode` and `enable_trace` are session variables

Actually I have no idea now what global variables we have now...